templates/serverpod_templates/projectname_server/Dockerfile

```diff
@@ -15,8 +15,9 @@ ENV role=monolith
 
 COPY --from=build /runtime/ /
 COPY --from=build /app/bin/server server
-COPY --from=build /app/config/ config/
-COPY --from=build /app/web/ web/
+COPY --from=build /app/confi[g]/ config/
+COPY --from=build /app/we[b]/ web/
+
```

Comment:
Suggestion: Add the migrations folder too!

Reply:
@Isakdl we dont have that currently:

https://github.com/serverpod/serverpod/blob/97d72a2ac377cc81a82a2e9f2ef6a82717283e12/templates/serverpod_templates/projectname_server_upgrade/Dockerfile#L1-L25

Should i add it still?

Reply:
@Isakdl done!